Dockerfile

```diff
@@ -0,0 +1,132 @@
+FROM node:12.14
+
+## ----------------------------------------------------------------------------
+## --- INSTALLING JAVA 11 -----------------------------------------------------
+## https://github.com/docker-library/openjdk/blob/master/11/jdk/Dockerfile
+## ----------------------------------------------------------------------------
```

Comment:
Installing JAVA 11 from openjdk Dockerfile. Everything is copied from their repository